manifests/z/Zoom/ZoomOutlookPlugin/5.15.5.926/Zoom.ZoomOutlookPlugin.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Zoom.ZoomOutlookPlugin
+PackageVersion: 5.15.5.926

```

```suggestion
PackageVersion: 5.15.5
```
This should match Control Panel